easybuild/easyconfigs/k/Keras/Keras-2.2.2-fosscuda-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'PythonPackage'
+
+name = 'Keras'
+version = '2.2.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://keras.io/'
+description = """Keras is a minimalist, highly modular neural networks library, written in Python and
+capable of running on top of either TensorFlow or Theano."""
+
+toolchain = {'name': 'fosscuda', 'version': '2018b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['468d98da104ec5c3dbb10c2ef6bb345ab154f6ca2d722d4c250ef4d6105de17a']
+
+dependencies = [
+    ('Python', '2.7.15'),
+    ('TensorFlow', '1.10.0', versionsuffix),
```

Comment:
@akesandgren Shall we bump this to TF 1.10.1? (to avoid being stuck dep-wise to TF 1.10.0 in `2018b` easyconfigs while we already have an easyconfig for TF 1.10.1...)

Reply:
Yeah, we should. I'll fix that.

Reply:
And in that case we should remove the 1.10.0-fosscuda from develop?

Reply:
I'm OK with either, since this is still in `develop`. I guess there's little point in keeping the 1.10.0 easyconfig for `fosscua/2018b`...

Reply:
The only reason would be that the patches are still called 1.10.0_...